### PR TITLE
Bump to 0.2.3-SNAPSHOT

### PR DESCRIPTION
**Pull request by densh**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+addSbtPlugin("me.lessis"    % "bintray-sbt" % "0.3.0")
+addSbtPlugin("com.jsuereth" % "sbt-pgp"     % "1.0.0")
```
Comment (sjrd) [inline project/plugins.sbt]: Is this change intended?

Comment (densh) [inline project/plugins.sbt]: Yep, previously sbt-pgp wasn't included in the build by default (i.e. no `publishSigned`.)
